packages/salesforcedx-vscode-soql/.vscodeignore

```diff
@@ -0,0 +1,19 @@
+.vscode/**
+.vscode-test/**
+out/test/**
+test/**
+src/**
+**/*.map
+.gitignore
+tsconfig.json
+
+# salesforcedx-utils-vscode files to ignore 
```

Comment:
These lines on `@salesforce/salesforcedx-utils-vscode` are only needed if you're planning on having that module as a dependency

Reply:
Got it, I'm not really sure if we will have the utils as a dependency, I don't think so at this time but I am not familiar with what is in that package, can you think of anything that might be useful to us?

Reply:
we have some general purpose utilities but i'm not sure you'll have to use them, we don't use them on all the other extensions. I think it's ok to remove this for now and add it back if needed.